playbooks/common/openshift-cluster/upgrades/etcd/backup.yml

```diff
@@ -3,6 +3,7 @@
   vars:
     embedded_etcd: "{{ groups.oo_etcd_to_config | default([]) | length == 0 }}"
     timestamp: "{{ lookup('pipe', 'date +%Y%m%d%H%M%S') }}"
+    etcdctl_command: "{{ 'etcdctl' if not openshift.common.is_containerized else 'docker exec etcd_container etcdctl' }}"
```

Comment:
I'm not sure that would work for embedded_etcd use for containerized installs, seems like a particular edge case that might cause problems.